### PR TITLE
Fixing stubbing issue in rest_spec

### PR DESCRIPTION
**Pull request by arjunmadan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
-  - "0.12"
+- "0.12"
 branches:
   only:
   - v1.0
+deploy:
```
Comment (jagoda) [inline .travis.yml:7]: This seems unrelated to the test cleanup.

**Comment by jagoda** [comment]
There seem to be a lot of changes listed here. It would be nice if this change was made in it's own branch so that only the relevant commits appear in the pull request.

File: .travis.yml (hunk 1 above)
Comment (arjunmadan) [inline .travis.yml:7]: This was when I was trying to setup Travis to deploy automatically to heroku. Should I rename the PR to mention this as well?
